src/ui/DynamicFacet/DynamicFacetRange.ts

```diff
@@ -52,6 +48,10 @@ export class DynamicFacetRange extends DynamicFacet implements IComponentBinding
   static parent = DynamicFacet;
   static doExport = () => exportGlobally({ DynamicFacetRange });
 
+  /**
+   * The options for the DynamicFacetRange
+   * @componentOptions
```

Comment:
Adding this will remove the "root" options for the DynamicFacet in the doc right?
You have any ideas how to fix that?